ckan/start_ckan_web_dev.sh

```diff
@@ -85,5 +87,18 @@ then
         echo
     done
 fi
+./create_datajson.sh &
 
-sudo -u ckan -EH paster serve --reload $CKAN_INI
+UWSGI_OPTS="--plugins http,python \
+            --socket /tmp/uwsgi.sock \
+            --wsgi-file /srv/app/wsgi.py \
+            --module wsgi:application \
+            --http 0.0.0.0:5001 \
+            --master --enable-threads \
+            --lazy-apps \
+            -p 2 -L -b 32768 --vacuum \
+            --harakiri $UWSGI_HARAKIRI"
+# Start uwsgi
+uwsgi $UWSGI_OPTS &
+nginx -g 'daemon off;'
+sleep 60
```

Comment:
Don't start up nginx here, and start uwsgi on port 5000

Reply:
dev mode is operational